src/party.cpp

```diff
@@ -419,7 +421,7 @@ void Party::shareExperience(uint64_t experience, Creature* source/* = nullptr*/)
 	leader->onGainSharedExperience(shareExperience, source);
 }
 
-bool Party::canUseSharedExperience(const Player* player) const
+bool Party::canUseSharedExperience(const Player* player)
```

Comment:
It's not a problem to have more code if this translates to better readability and quality.